### DISCUSSION TITLE
Adding simple elements to a so.Plot

### DISCUSSION THREAD

**Discussion by danikolbe** [mwaskom/seaborn#3420] [Q&A]
Hi,
here's a simple question which I haven't quite figured out yet. How do I add something as a simple as a horizontal line, or say a box of lines into an so.Plot? This would usually be done by plt.axhline if I am not mistaken, but this doesn't work for seaborn objects. Any ideas? I'm sure there's an easy way I am not seeing. Thanks!
Best,
Dan

**Comment by danikolbe** [comment]
Current solution:

```
p = (
    so.Plot()
)


f = mpl.figure.Figure()
p.on(f).plot()

ax = f.axes[0]
ax.axhline(y=1.3, color='blue', linestyle='-.', alpha=0.5, linewidth=0.5)

```

**Reply by thuiop** [reply]
You can also do `.on(ax)` and directly provide an Axes instance instead, although the behaviour is a bit different.

**Comment by mwaskom** [comment]
There's not currently support for this through the objects interface, though I do plan to add it.
